music21/stream/__init__.py

```diff
@@ -4029,8 +4030,12 @@ def hasMeasureNumberInformation(measureIterator):
         # used coreInsert
         returnObj.coreElementsChanged()
 
-        if gatherSpanners:
-            returnObj.coreGatherMissingSpanners()
+        if gatherSpanners:  # True, GatherSpanners.ALL, or GatherSpanners.COMPLETE_ONLY
+            requireAllPresent = (gatherSpanners == GatherSpanners.COMPLETE_ONLY)
```

Comment:
Thanks!  I'm writing a lot of cases for backwards compatibility, where I want to compare == instead of `is`, but this is NOT one of them (since "COMPLETE_ONLY" is the new flag that didn't exist in True/False)